benchmark/inference/hetero_sage.py

```diff
@@ -0,0 +1,49 @@
+import torch
+from tqdm import tqdm
+
+from torch_geometric.nn import SAGEConv, to_hetero
+
+
+class HETERO_SAGE:
```

Comment:
```suggestion
class HeteroGraphSAGE(torch.nn.Module):
```
I think we should just make this a module.